MesaModel/hamilton/example_slurm_scripts/batchrunner_with_six_classes.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --ntasks 1
+#SBATCH --cpus-per-task 24
```

Comment:
Should we be running on 24 CPUs if there are only 6 classes, so only 6 jobs in the pathos queue?